### PR TITLE
User app

### PR DESCRIPTION
**Pull request by BorisBesker**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Django (1.10.6)
+django-debug-toolbar (1.7)
+MySQL-python (1.2.5)
+mysqlclient (1.3.10)
+pip (9.0.1)
```
Comment (marjev) [inline requirements.txt]: Pip shouldn't be in this file.
This file is used by pip to install packages by calling `pip install -r requirements.txt`.

Data should be formatted the same way the output of `pip freeze` command is.

Comment (BorisBesker) [inline requirements.txt]: Done.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Django==1.10.6
+django-debug-toolbar==1.7
+MySQL-python==1.2.5
+mysqlclient==1.3.10
```
Comment (marjev) [inline requirements.txt]: Are you using both `MySQL-python` and `mysqlclient`?

Comment (BorisBesker) [inline requirements.txt]: Yes, unfortunately server won't run if both of these packages aren't installed.

**Pull request merged**